### PR TITLE
Service portal/mvp ui

### PR DESCRIPTION
**Pull request by atligudl**
# \<UI updates to Service Portal\>

## What
UI components updated
UI simplified
Mock components removed
Localization work
Responsive styles
Mobile toolbar
Menu component
Mobile Menu

## Why
Prepare service portal for mvp deployment

## Screenshots / Gifs
2020-10-07 12_52_05-Movies   TV
2020-10-07 12_51_44-Mitt Ísland
2020-10-07 12_52_56-Mitt Ísland
2020-10-07 12_52_43-Mitt Ísland
2020-10-07 12_52_26-Mitt Ísland

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against master before asking for a review

### CODE REVIEW

**Review by dabbeg — DISMISSED** [review]
:+1:

**Review by Thorrik58 — APPROVED** [review]
LGTM, only thing I was wondering if we want to make the HexToRGB function reusable in the ui library as a util.

File: libs/service-portal/family/src/screens/UserInfo/UserInfo.tsx [hunk 1]
```diff
@@ -9,9 +9,11 @@ import {
 } from '@island.is/island-ui/core'
 import { ServicePortalModuleComponent } from '@island.is/service-portal/core'
 import UserInfoLine from '../../components/UserInfoLine/UserInfoLine'
-import { useNatRegGeneralLookup } from '@island.is/service-portal/graphql'
-import UserInfoSidebars from './UserInfoSidebars'
+// import { useNatRegGeneralLookup } from '@island.is/service-portal/graphql'
```
Comment (arnifreyrs) [inline libs/service-portal/family/src/screens/UserInfo/UserInfo.tsx:12]: consider cleaning up these commented out lines if they are unnecessary

**Review by arnifreyrs — APPROVED** [review]
It is hard to provide meaningful feedback on such a huuuuuge PR, but overall it looks really nice, and I am excited to see the new my pages. 

One thing I am wondering about, is that the image files. We will definitely need some of these images for the parental leave application and other applications. We might need to think more about where we store these images in our repo, and make them accessible to other projects.

File: libs/service-portal/family/src/screens/UserInfo/UserInfo.tsx (hunk 1 above)
Comment (atligudl) [reply libs/service-portal/family/src/screens/UserInfo/UserInfo.tsx:12]: Yup sorry about the size, had some difficulties in compartmentalizing each update

I agree with the images, maybe something we should bring up in tomorrow's discipline meeting

**Comment by atligudl** [comment]
> LGTM, only thing I was wondering is do we want to make the HexToRGB function reusable in the ui library as a util?

Thought about that too, lets close this PR before it gets any bigger and bring the issue about reusable style util functions to other updates :)

**Pull request merged**